paddle/operators/math/math_function.cc

```diff
@@ -130,6 +130,87 @@ void matmul<platform::CPUPlace, double>(
       matrix_b.data<double>(), beta, matrix_out->data<double>());
 }
 
+#ifdef PADDLE_USE_MKLML
```

Comment:
Is it a good idea to move `batched_gemm` into `gemm.{cc,cu}` or `batched_gemm.{cc,cu}` so to prevent math_function.cc from explode?

Reply:
I was looking into this just now, but I think the `batched_gemm` code is very closely related to the `gemm` code. They need the same includes, etc., and if one needs to be modified, very likely also the other. I think it makes sense to keep them together. But please let me know if you still prefer that I split them.